### ISSUE TITLE
我是看了你的博客，找到你的，你的代码位置是不是不对。                                              

### ISSUE THREAD

**Issue by dtzfast** [Xingxiangrui/various_loss_and_intermedia_supervision#1]
def forward(self, x):
        # [B_batch,N_nodes,C_channels]
        B, N, C = x.size()  ## 因为先拿出数据尺寸，下面再乘一下。
        # h = torch.bmm(x, self.W.expand(B, self.in_features, self.out_features))  # [B,N,C]
        h = torch.matmul(x, self.W)  # [B,N,C]
        a_input = torch.cat([h.repeat(1, 1, N).view(B, N * N, C), h.repeat(1, N, 1)], dim=2).view(B, N, N,
                                                                                                  2 * self.out_features)  # [B,N,N,2C]
        # temp = self.a.expand(B, self.out_features * 2, 1)
        # temp2 = torch.matmul(a_input, self.a)
        attention = self.leakyrelu(torch.matmul(a_input, self.a).squeeze(3))  # [B,N,N]
 
        attention = F.softmax(attention, dim=2)  # [B,N,N]
        attention = F.dropout(attention, self.dropout, training=self.training)
        h_prime = torch.bmm(attention, h)  # [B,N,N]*[B,N,C]-> [B,N,C]
        out = F.elu(h_prime + self.beta * h)
        return out
————————————————
版权声明：本文为CSDN博主「祥瑞Coding」的原创文章，遵循CC 4.0 BY-SA版权协议，转载请附上原文出处链接及本声明。
原文链接：https://blog.csdn.net/weixin_36474809/article/details/89401552
